### DISCUSSION TITLE
De novo kmer assembly in ArchR

### DISCUSSION THREAD

**Discussion by Y-ogativity** [GreenleafLab/ArchR#1165] [Feature Requests]
Hi, Thank you for developing such great scATAC-seq software. I wonder whether, in ArchR environment, I could execute de-novo motif finding as is implemented by [chromVAR](https://greenleaflab.github.io/chromVAR/articles/Articles/Applications.html).

**Comment by rcorces** [comment]
Unfortunately, this is not currently possible in ArchR. Though it is possible to export peak sets and perform de novo motif finding in other tools.

**Reply by Y-ogativity** [reply]
Thank you for the reply. I will try some.
